packaging/windows/variables.wxi

```diff
@@ -26,7 +26,7 @@
   <?if $(var.Platform)=x86?>
   <?define Program_Files="ProgramFilesFolder"?>
   <?define Win64AttributeValue=no?>
-  <?define UpgradeCode="70A1576F-63B6-4659-9E39-25C7B769DDE5"?>
+  <?define UpgradeCode="25CAE344-22A6-422D-942E-942687C56A45"?>
```

Comment:
Uhh, wait. Do you want to be able to install both x86 and x64 MSIs at the same time?

If you change the GUID for x64 then it won't upgrade older versions of the MSI and we'll go through the whole thing of old versions not being uninstalled again (like with the version mess from before).